### DISCUSSION TITLE
part-ii/2017-98/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#413] [Question comments]
Part II 2017 Number Fields: Paper 4, Section II, H 

https://questions.tripos.org/part-ii/2017-98/

**Comment by artie2000** [comment]
(a): bookwork

<details>
<summary>(b)</summary>

Relative norms and traces give $\mathcal{O}_K\subseteq \mathbb{Z}[\sqrt2,\sqrt{d}]+\frac{1}{2}\mathbb{Z}[\sqrt{2}+\sqrt{2d}]$.

Then $\frac{\sqrt2+\sqrt{2d}}{2}$ is in fact an algebraic integer, so an integral basis is

$(1,\sqrt2,\sqrt{d},\frac{\sqrt2+\sqrt{2d}}{2})$

</details>
<details>
<summary>(c)</summary>

$C_4$ (generated by a factor of $(3)$)

</details>
